packages/client/hmi-client/src/components/widgets/tera-expandable-panel.vue

```diff
@@ -27,8 +27,9 @@ import InputSwitch from 'primevue/inputswitch';
 const emit = defineEmits(['delete', 'edit', 'update:is-included']);
 
 defineProps<{
-	hideDelete: boolean;
+	hideDelete?: boolean;
 	isIncluded?: boolean;
+	hideEdit?: boolean;
```

Comment:
Positive boolean name is always a more readable choice.
```suggestion
	isDeletable?: boolean;
	isIncluded?: boolean;
	isEditable?: boolean;
```

Reply:
done